TestbedScripts/TestEnvUtils.ps1

```diff
@@ -43,11 +43,57 @@ function Initialize-DockerNetwork {
     }
 }
 
-function Initialize-ContainerInterface {
+function Get-MacOfContainer {
     Param ([Parameter(Mandatory = $true)] [string] $ContainerName)
-    Write-Host $("TODO: Initializing network interface for container " +`
-        "'$ContainerName'...")
-    # TODO JW-883
+    $mac = docker exec $ContainerName powershell -Command {
+        (Get-NetAdapter | Select-Object -first 1).macaddress
+    }
+    return $mac
+}
+
+function Initialize-ContainerInterface {
+    Param ([Parameter(Mandatory = $true)] [string] $ContainerName,
+           [Parameter(Mandatory = $true)] [string] $ContainerIP,
+           [Parameter(Mandatory = $true)] [string] $PrefixLength,
+           [Parameter(Mandatory = $true)] [string] $TheOtherMac,
+           [Parameter(Mandatory = $true)] [string] $TheOtherIP)
+    $command = ('$ifname = (Get-NetAdapter | Select-Object -first 1).Name; ' +`
+        '$ip_set_properly = (New-NetIPAddress -interfacealias ' +`

```

Comment:
`ip_set_properly` name might be misleading. It could be named `ip_addresses_count`